docker/README.md

```diff
@@ -0,0 +1,39 @@
+# Docker Information
+
+Developer Dockerfile is used to set up the development environment.
+It includes all the necessary tools, libraries, and dependencies required
+for development. This ensures that all developers work in a consistent
+environment, reducing the "it works on my machine" problem.
+Documentation Dockerfile is used to build and serve your project's documentation.
```

Comment:
I don't think it is currently able to serve the documentation. If I am right, please remove `and serve`